README.md

```diff
@@ -146,6 +146,15 @@ Semantic HTML is the use of HTML markup to reinforce the semantics, or meaning,
 * Explain event delegation
 * Explain how `this` works in JavaScript
 * Explain how prototypal inheritance works
+    Each constructor instance inherits all its methods and properties.
```

Comment:
prototypal inheritance means prototypes of objects are linked, which means, that when some property is used it is first searched at the object, then (when not found) on its prototype and so on